app/Http/Middleware/ApiAuth.php

```diff
@@ -10,15 +10,15 @@ class ApiAuth
     /**
      * Handle an incoming request.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Closure  $next
+     * @param  \Illuminate\Http\Request $request
+     * @param  \Closure $next
      *
      * @return mixed
      */
     public function handle($request, Closure $next)
     {
-        if (request()->header('token') != config('handesk.api_token')) {
-            return response()->json(['error' => 'unauthorized'], Response::HTTP_FORBIDDEN);
+        if (request()->header('token') && (request()->header('token') != config('handesk.api_token'))) {
```

Comment:
I believe this can be unsafe as if the user doesn't send the token header, it won't get the unauthorized response

Reply:
but won't you be able to use the api if you don't send any token?

Reply:
Yeah.. that's true, Will send another PR that corrects it. I think its better setting ApiAuth middleware in the route group that requires it..

Reply:
middleware in the route group that corrects it sounds ok to me

Reply:
Can you fix that no token authorization first so I can merge this one?
Thank you!

Reply:
Corrections has been made in my latest commit https://github.com/BadChoice/handesk/pull/158/commits/31aed5d6d0716ba1f0a312b88b63ffb4e3b08e85